src/plugins/process/bias_correction/medItkBiasCorrectionProcess.cpp

```diff
@@ -0,0 +1,516 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#include <medItkBiasCorrectionProcess.h>
+
+#include <dtkLog>
+
+#include <itkImage.h>
+#include <itkBinaryBallStructuringElement.h>
+#include <itkCommand.h>
+
+#include <medAbstractImageData.h>
+#include <medAbstractDataFactory.h>
+
+#include <medIntParameter.h>
+#include <medDoubleParameter.h>
+#include <medStringParameter.h>
+
+#include <itkN4BiasFieldCorrectionImageFilter.h>
+#include <itkExtractImageFilter.h>
+#include <itkConstantPadImageFilter.h>
+#include <itkOtsuThresholdImageFilter.h>
+#include <itkShrinkImageFilter.h>
+#include <itkTimeProbe.h>
+
+#define MAX_INT_POSITIVE 4294967295
```

Comment:
There is already INT_MAX or std::numeric_limits http://www.cplusplus.com/reference/limits/numeric_limits/ . No need to redefine one